.github/workflows/module-tests.yml

```diff
@@ -51,6 +51,17 @@ jobs:
           if [ "${AWS_ACCESS_KEY_ID}" != "" ]; then
             ./mvnw test -B -Dair.check.skip-all -pl presto-hive -P test-hive-glue
           fi
+      - name: Run Hive Azure ABFS Tests
+        if: matrix.config != 'config-empty'
```

Comment:
Why? Is your intention to run this for  `config-cdh5`? (upcoming; already mentioned in this file)

Add a comment. Maybe

```
# config-empty's Hive 1.2 does not support Azure storage
```